src/cdk/overlay/position/flexible-connected-position-strategy.ts

```diff
@@ -839,6 +842,17 @@ export class FlexibleConnectedPositionStrategy implements PositionStrategy {
       overlayPoint = this._pushOverlayOnScreen(overlayPoint, this._overlayRect);
     }
 
+    // @deletion-target 7.0.0 Remove `_overlayContainer` null check
+    // once it's guaranteed to be defined.
```

Comment:
Could you expand this to explain the circumstances under which `overlayContainer` is not defined?

Reply:
I've updated the comment to be a bit more thorough.